### ISSUE TITLE
Hi L4ki KDE help

### ISSUE THREAD

**Issue by tjw0lf** [L4ki/Amethyst#1]
Was wondering if you can help me out. How do I change the menu color in KDE

**Comment by L4ki** [comment]
Hi, tjw0lf!
I assume you mean GTK applications in KDE Plasma?
If so, then download GTK-2/3 Theme "Amethyst" in: System Settings → Application Style → GNOME / GTK Application Style

Bottom Right: Get New GNOME / GTK Application Styles
Next: Select a GTK3 theme drop down menu, called "Amethyst"
Greeting,

**Comment by tjw0lf** [comment]
Hi L4ki

I mean the highlight color in the menu I found out its one of the widgets svgz But I do not know which one. Which program do you use to tell.

Thanks for help

**Comment by L4ki** [comment]
I use the program: Inkscape

**Comment by tjw0lf** [comment]
Cool thats what i using. 

How do i tell which one is the highlight color for the menu Application Menu I pretty much opened all of them but inkscape didn't show color.  do I need to set special settings. I'm trying to change highlight color in the menu i know its svgz image but do not know which one

**Comment by L4ki** [comment]
Реците ми конкретно коју тему тражите: плазма тема 'Аметист', ГТК тема?
Имам и 'Аметист-Лигхт'?

**Comment by tjw0lf** [comment]
I'm using the breath Theme by Manjaro. I like it but I'm trying to change the Menu Highlight color from green to purple. I wanted to know how you did it in Inkscape

**Comment by L4ki** [comment]
Ok, can you give me a screenshoot, what exactly do you want to change from green to purple?

**Comment by L4ki** [comment]
To get started: In Folder widgets, open viewitem.svg or svgz, with Inkscape

**Comment by tjw0lf** [comment]
Here you go. its that green highlight hover.

<a href="https://imgur.com/1ps9Nmd"><img src="https://i.imgur.com/1ps9Nmd.png" title="source: imgur.com" /></a>

**Comment by tjw0lf** [comment]
Thanks so much for taking time to help me You're Awesome :)

**Comment by L4ki** [comment]
I'm glad I helped you:)

**Comment by tjw0lf** [comment]
I have another question I see a couple of these with the background color on them How do you know which one goes where. Do change the background color.

<a href="https://imgur.com/0iYvkM8"><img src="https://i.imgur.com/0iYvkM8.png" title="source: imgur.com" /></a>

**Comment by L4ki** [comment]
No, with this second icon on the left, click on this black widget
But this is not the widget you asked me for.
Open the widget viewitem.svgz

https://i.postimg.cc/kG9C2q60/Screenshot-20190925-202754.png

**Comment by tjw0lf** [comment]
I know I was asking about these backgrounds ones.
